Tmain/readtags-combine.d/run.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/sh
 
-# Copyright: 2016 Masatake YAMATO
-# License: GPL-2
-
```

Comment:
Please, add YOUR copyright notice instead.

Reply:
Please, squash 450c297 and ff70ecf into one comit, and do "git push --force ...".

Reply:
Would the copyright notice be that of the company I am working for (CCI Europe) or just me personally?
We are using ctags to be able to easily browse our code using the vim editor.

My suggestion would be something like the below. Please advise, I am new to open source contribution. Thanks in advance!

# Copyright: 2019 CCI Europe. Author: Claus Moltke-Leth
# License: GPL-2

Reply:
It may be up to the relation between your and your company.

Did you write the run.sh as part of your job?
In that case, the copyright notice you showed looks good for me.

Reply:
I have included copyright information now. As far as I am concerned, the pull request is complete. Should I close it, or will you do it?